### ISSUE TITLE
[Outreachy Task Submission] Accessibility Issue with the 'main-filter toggle' button value text in map/data screens

### ISSUE THREAD

**Issue by Chiemezuo** [ushahidi/platform#4905]
# Intro
This is an Accessibility issue I found with the 'main filter' toggle button in the Map/Data screen. It is from the arrow button that shows or hides surveys and sources, depending on the toggle state.

## Issue
The toggle button for `main-filter` has no value text. ***This is a violation of WCAG Success Criterion 1.1.1 (Level A) - Non-text content***. [See](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html)

## Description.
***Success Criterion 1.1.1 (Level A) - Non-text content*** requires that all non-text content presented to users has a text alternative that serves the equivalent purpose, except for situations where the content is intended to be omitted by the author or is decorative in nature.

## Impact
Accessibility software that can either convert text to visuals (for deaf people), and braille (for blind people) will not work. People who cannot understand the meaning of images/icons would struggle with this. Additionally, there would be no way to search for visual content in a text-based manner. 

## Fix
Add a value text to the button. I will raise a PR for this, shortly.
